opus/import/obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,246 @@
+# Defines the ObsBundleCassiniUvisSolarOccBeckerJarmak class, which encapsulates
+# fields in the common, common occultation, obs_mission_cassini, and obs_instrument_couvis
+# tables for COUVIS solar occultations. This class supports derived data from UVIS EUV
+# archived in cassini_uvis_solarocc_beckerjarmak2023.
+################################################################################
+
+from datetime import datetime, date
+from obs_bundle_occ_common import ObsBundleOccCommon
+from obs_cassini_common_pds4 import ObsCassiniCommonPDS4
+
+class ObsBundleCassiniUvisSolarOccBeckerJarmak(ObsBundleOccCommon, ObsCassiniCommonPDS4):
+    def __init__(self, *args, **kwargs):
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)